content/BCBestPractices/Readability/line-start-keywords/index.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "Line Start Keywords"
+weight = 740
++++
+
+<_Created by Microsoft, Described by waldo_\>
+
+## Description
+The `end`, `if`, `repeat`, `for`, `while`, `else` and `case` statement should always start a line. 
+
+## Bad code
+
+```al
+    if IsContactName then ValidateContactName
+    else if IsSalespersonCode then ValidateSalespersonCode
+    else if IsSalesCycleCode then ValidatSalesCycleCode;
+```
+
+## Good code
+
+```al
+    if IsContactName then
+        ValidateContactName
+    else
+        if IsSalespersonCode then
+            ValidateSalespersonCode
+        else
+            if IsSalesCycleCode then ValidatSalesCycleCode;
+```
+
```

Comment:
Why is the statement after the then on the last last not on a line by itself, while it is on the other lines?
Shouldn't it be:

```al
    if IsContactName then
        ValidateContactName
    else
        if IsSalespersonCode then
            ValidateSalespersonCode
        else
            if IsSalesCycleCode then 
                ValidatSalesCycleCode;
```

Reply:
Nice catch again :-)